tests/Integration/ResourceStatusSystem/Test_ResourceManagement.py

```diff
@@ -27,15 +30,15 @@ def test_addAndRemove():
   # ...............................................................................
 
   res = rsClient.addOrModifyAccountingCache('TestName12345', 'plotType', 'plotName', 'result', dateEffective, lastCheckTime)
-  assert res['OK'] == True
+  assert res['OK'] is True
```

Comment:
This will not work. All the native ```assert``` statement are ignored with the python we distribute because it is compiled with -O